install_script.sh.template

```diff
@@ -795,50 +826,58 @@ fi
-stop_instructions="$sudo_cmd $service_cmd $system_service stop"
-start_instructions="$sudo_cmd $service_cmd $system_service start"
-
-if [[ `$sudo_cmd ps --no-headers -o comm 1 2>&1` == "systemd" ]] && command -v systemctl 2>&1; then
-  # Use systemd if systemctl binary exists and systemd is the init process
-  restart_cmd="$sudo_cmd systemctl restart ${system_service}.service"
-  stop_instructions="$sudo_cmd systemctl stop $system_service"
-  start_instructions="$sudo_cmd systemctl start $system_service"
-elif /sbin/init --version 2>&1 | grep -q upstart; then
-  # Try to detect Upstart, this works most of the times but still a best effort
-  restart_cmd="$sudo_cmd stop $system_service || true ; sleep 2s ; $sudo_cmd start $system_service"
-  stop_instructions="$sudo_cmd stop $system_service"
-  start_instructions="$sudo_cmd start $system_service"
-fi
-
-if [ $no_start ]; then
-    printf "\033[34m
-* DD_INSTALL_ONLY environment variable set: the newly installed version of the
-$nice_flavor will not be started. You will have to do it manually using the
-following command:
-
-    $start_instructions
-
-\033[0m\n"
-    exit
-fi
-
-printf "\033[34m* Starting the $nice_flavor...\n\033[0m\n"
-eval "$restart_cmd"
-
-
-# Metrics are submitted, echo some instructions and exit
-printf "\033[32m
-
-Your $nice_flavor is running and functioning properly. It will continue
-to run in the background and submit metrics to Datadog.
-
-If you ever want to stop the $nice_flavor, run:
-
-    $stop_instructions
-
-And to run it again run:
-
-    $start_instructions
+#
+declare -a services
+services=("$system_service")
+if [ -n "$fips_mode" ]; then
+  services+=("$system_fips_service")
+fi
+
+preamble="  * DD_INSTALL_ONLY environment variable set."
+for current_service in "${services[@]}"; do
+  restart_cmd="$sudo_cmd $service_cmd $current_service restart"
+  stop_instructions="$sudo_cmd $service_cmd $current_service stop"
+  start_instructions="$sudo_cmd $service_cmd $current_service start"
+
+  if [[ `$sudo_cmd ps --no-headers -o comm 1 2>&1` == "systemd" ]] && command -v systemctl 2>&1; then
+    # Use systemd if systemctl binary exists and systemd is the init process
+    restart_cmd="$sudo_cmd systemctl restart ${current_service}.service"
+    stop_instructions="$sudo_cmd systemctl stop $current_service"
+    start_instructions="$sudo_cmd systemctl start $current_service"
+  elif /sbin/init --version 2>&1 | grep -q upstart; then
+    # Try to detect Upstart, this works most of the times but still a best effort
+    restart_cmd="$sudo_cmd stop $current_service || true ; sleep 2s ; $sudo_cmd start $current_service"
+    stop_instructions="$sudo_cmd stop $current_service"
+    start_instructions="$sudo_cmd start $current_service"
+  fi
 
-\033[0m"
+  if [ $no_start ]; then
+    printf "\033[34m\n${preamble}
+    The newly installed version of the ${flavor_to_readable[$current_service]} will not be started.
```

Comment:
You can define a variable containing `${flavor_to_readable[$current_service]}` at the start of the loop, since it's reused multiple times.